package.json

```
@@ -21,9 +21,9 @@
     "dependencies": {
         "@babel/core": "^7.16.0",
         "@babel/eslint-parser": "^7.16.0",
-        "@hapi/bossy": "5.x.x",
+        "@hapi/bossy": "^6.0.0",
         "@hapi/eslint-plugin": "^5.1.0",
```

Pending https://github.com/hapijs/eslint-plugin/pull/31
```suggestion
        "@hapi/eslint-plugin": "^6.0.0",
```